.drone.windows.yml

```diff
@@ -1,30 +1,57 @@
 ---
 kind: pipeline
-name: windows-amd64
+name: testing
 
 platform:
   os: windows
   arch: amd64
+  version: 1803
+
+steps:
+- name: vet
+  pull: always
+  image: golang:1.11-windowsservercore-1803
+  commands:
+  - go vet ./...
+  environment:
```

Comment:
Still missing a volume?

Reply:
Added logic for windows volume with golang